src/atomate2/openmm/utils.py

```diff
@@ -2,16 +2,28 @@
 
 from __future__ import annotations
 
+import io
 import re
 import tempfile
 import time
 import warnings
 from pathlib import Path
 from typing import TYPE_CHECKING
 
+import numpy as np
+import openmm.unit as omm_unit
+from emmet.core.openmm import OpenMMInterchange
+from openmm import LangevinMiddleIntegrator, XmlSerializer
+from openmm.app import PDBFile
+
 if TYPE_CHECKING:
+    from contextlib import suppress
+
     from emmet.core.openmm import OpenMMTaskDocument
 
+    with suppress(ImportError):
```

Comment:
`TYPE_CHECKING` is always `False` at run time so no need to `suppress` errors here. also, `suppress` is quite slow compared to `try/except` so would usually opt for that

Reply:
I initially did try/except but it was caught by the linter. should I override then?

Reply:
yes, we should ignore that `ruff` rule that recommends `contextlib.suppress`. bad rule imo